### PR TITLE
chore: update the log when failed to get schema for plugin

### PR DESCRIPTION
**Pull request by Hazel6869**
### Description

Fixes # (https://github.com/apache/apisix/issues/8653)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: apisix/plugin.lua [hunk 1]
```diff
@@ -926,7 +926,7 @@ _M.decrypt_conf = decrypt_conf
 local function encrypt_conf(name, conf, schema_type)
     local schema = get_plugin_schema_for_gde(name, schema_type)
     if not schema then
-        core.log.warn("failed to get schema for plugin: ", name)
+        core.log.warn("make sure you enable the plugin, failed to get schema for plugin: ", name)
```
Comment (tokers) [inline apisix/plugin.lua:929]: What's the meaning of this change?

**Comment by tokers** [comment]
@Hazel6869 Please see comments in https://github.com/apache/apisix/issues/8653 carefully. The way suggested is by no means of updating the error message.

**Comment by Hazel6869** [comment]
> @Hazel6869 Please see comments in #8653 carefully. The way suggested is by no means of updating the error message.

ok

**Comment by monkeyDluffy6017** [comment]
I will close this PR as it has not been active for a long time

**Pull request closed without merging**